node/src/types.rs

```diff
@@ -133,3 +136,52 @@ impl ReshareProtocol {
         self.protocol.write().await
     }
 }
+
+#[derive(Copy, Clone, Debug)]
+pub struct LatestBlockHeight(pub near_primitives::types::BlockHeight);
+
+impl LatestBlockHeight {
+    pub async fn fetch(gcp: &GcpService) -> DatastoreResult<Self> {
+        gcp.datastore.get("latest").await
```

Comment:
how does making name_key="latest" get the latest block height from datastore?

Reply:
this is a singular entry in LatestBlockHeight-{env}. We will always reference the "latest" entity